.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: "\U0001F41B Bug report"
+about: "If something isn't working \U0001F622"
+title: ''
+labels: Bug
+assignees: ''
+
+---
+
+<!-- Please provide the following information: -->
+
+### Summary
+<!-- Please provide a brief outline of the issue -->
+
+### Description
+<!-- Provide detailed information about this issue, including: -->
+* Which specific command was run? (Example: `ecs-cli up --capability-iam --size 2 --launch-type EC2`)
+* Which version of the CLI you are using? (Run: `ecs-cli --version`)
+* Which version of Go are you using? (Run: `go version`)
+* What platform are you using to run ECS CLI commands? (E.g. Linux, MacOSX, Windows)
+
+### Config files
+<!--
+For `compose` or `compose service` commands
+Please note that GitHub issues are public, remove sensitive data from your config files before posting.
+If you are not comfortable posting your files here, please let us know and we can provide an alternate method.
+-->
+* docker-compose.yml
+
+* ecs-params.yml
+
+* ~/.ecs/config
+
+<!-- Not required for feature requests -->
```

Comment:
Do we need this comment now that we have a separate template?

Reply:
Removed 👍